dts/arm/realtek/ec/rts5912-pinctrl.dtsi

```diff
@@ -0,0 +1,645 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ *
+ */
+
+#include <zephyr/dt-bindings/pinctrl/realtek-rts5912-pinctrl.h>
+
+&pinctrl
+{
+	/* ADC PINCTRL SETTING START */
+	/omit-if-no-ref/ adc0_gpio074: adc0_gpio074 {
+			pinmux = <REALTEK_RTS5912_PINMUX(74, REALTEK_RTS5912_FUNC1)>;
+			input-enable;
+			input-schmitt-enable;
```

Comment:
why do all entries default to input-enable, input-schmitt-enable?

Reply:
In our situation, GPIO in input mode is more suitable for not affecting other devices which connect to our pins.
`input-schmitt-enable` is needed because hysteresis is used in all pins. Thanks.